rta/src/main/java/com/gluonhq/richtextarea/RichListCell.java

```diff
@@ -353,4 +359,11 @@ private Optional<ParagraphTile> getParagraphTile() {
         return Optional.empty();
     }
 
+    private static Color parseColorOrDefault(String color, Color defaultColor) {
+        try {
+            return Color.web(color);
+        } catch (Exception e) {
+            return defaultColor;
```

Comment:
Shouldn't a log entry explain here what was being parsed? I can imagine users wondering why they select RED and are returned with the defaultColor.

Reply:
`Color::web` admits different formats for colours, like `"#ff0000"` or `"0xff0000"` or `red` or `rgb(255,0,0)`, (see https://github.com/openjdk/jfx/blob/master/modules/javafx.graphics/src/main/java/javafx/scene/paint/Color.java#L288).

But if there are typos or not valid strings, like unknown colours, there would be an IllegalArgumentException exception, and that would be avoided by using a default colour.